css/style.css

```diff
@@ -6,3 +6,198 @@
  *    for example: General styles, Navigation styles, Hero styles, Footer etc.
  * 
  */
+/*******************************************/
+ /*---- COLOR SCHEMES for TEXTS AND LINKS---Konika*/
+/*******************************************/
+
+*{box-sizing:border-box;
+  padding: 0;
+  margin:0;
+}
+body{
+    background-color:#e8e1e0 ;
+}
+h1,h2,h3{
+    color:#57394a
+}
```

Comment:
Take care with the formatting of the CSS file - you should include a space between the selector and the first curly brace, indent rules within a block etc. It is also a good idea to include a new line between rules to aid readability.

You can auto format CSS in VSCode to fix much of this;

On Windows Shift + Alt + F.
On Mac Shift + Option + F.
On Linux Ctrl + Shift + I.

Reply:
Hi Paul, Thank you for your comment. I addressed the issue and edited the formatting of the CSS file.